tests/acceptance/features/apiNotification/spaceNotification.feature

```diff
@@ -301,132 +126,54 @@ Feature: Notification
-                  },
-                  "name": {
-                    "type": "string",
-                    "enum": [
-                      "notification checking"
-                    ]
-                  }
+                  "id": {"pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}\\$[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}![a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"},
+                  "name": {"const": "notification checking"}
                 }
               },
               "user": {
                 "type": "object",
-                "required": [
-                  "displayname",
-                  "id",
-                  "name"
-                ],
+                "required": ["displayname","id","name"],
                 "properties": {
-                  "displayname": {
-                    "type": "string",
-                    "enum": [
-                      "Alice Hansen"
-                    ]
-                  },
-                  "id": {
-                    "type": "string",
-                    "pattern": "^%user_id_pattern%$"
-                  },
-                  "name": {
-                    "type": "string",
-                    "enum": [
-                      "Alice"
-                    ]
-                  }
+                  "displayname": {"const": "Alice Hansen"},
+                  "id": {"pattern": "^%user_id_pattern%$"},
+                  "name": {"const": "Alice"}
                 }
               }
             }
           },
-          "notification_id": {
-            "type": "string"
-          },
-          "object_id": {
-            "type": "string",
-            "pattern": "^%space_id_pattern%$"
-          },
-          "object_type": {
-            "type": "string",
-            "enum": [
-              "storagespace"
-            ]
-          },
-          "subject": {
-            "type": "string",
-            "enum": [
-              "Space disabled"
-            ]
-          },
-          "subjectRich": {
-            "type": "string",
-            "enum": [
-              "Space disabled"
-            ]
-          },
-          "user": {
-            "type": "string",
-            "enum": [
-              "Alice"
-            ]
-          }
+          "notification_id": {"type": "string"},
+          "object_id": {"pattern": "^%space_id_pattern%$"},
+          "object_type": {"const": "storagespace"},
+          "subject": {"const": "Space disabled"},
+          "subjectRich": {"const": "Space disabled"},
+          "user": {"const": "Alice"}
         }
       }
       """
 
 
-  Scenario Outline: get a notification about a space share in various languages
+  Scenario Outline: get in-app notification about a space share in various languages
```

Comment:
what about email notifications?

Reply:
in these scenario we send share request before changing language, mail will be triggered in default language so no point  to tests mail. Below there is a scenario which check mail and in-app notification for group share